modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getWebContents.js

```diff
@@ -17,66 +18,73 @@ import {
 
 export function getWebContents({dxpVersion, slaCurrent, subscriptionGroups}) {
 	const webContents = [];
+	const hasProjectSLA = Object.values(SLA_TYPES).some((slaType) =>
+		slaCurrent?.includes(slaType)
+	);
 
-	if (
-		subscriptionGroups.some(
-			({name}) =>
-				name === PRODUCT_TYPES.dxp ||
-				name === PRODUCT_TYPES.portal ||
-				name === PRODUCT_TYPES.commerce
-		) ||
-		!subscriptionGroups.some(
-			({name}) =>
-				name === PRODUCT_TYPES.partnership ||
-				name === PRODUCT_TYPES.dxpCloud
-		)
-	) {
+	const initialSubscriptions = Object.keys(
+		PRODUCT_TYPES
+	).map((productKey) => [productKey, false]);
+
+	const hasSubscriptionGroup = subscriptionGroups?.reduce(
+		(subscriptionGroupsAccumulator, subscriptionGroup) => {
+			const [currentProduct] = Object.entries(PRODUCT_TYPES).find(
+				([, productName]) => productName === subscriptionGroup?.name
+			);
+
+			if (currentProduct) {
+				subscriptionGroupsAccumulator[currentProduct] = true;
+			}
+
+			return subscriptionGroupsAccumulator;
+		},
+		Object.fromEntries(initialSubscriptions)
+	);
```

Comment:
Hi @iagochaves, as we discussed before. Let's try to avoid the use of Object API, to improve our code readability, simplify the approach and remove dependencies between variables as constants and/or keys that can change :)

As we have a team with constant ramp up, we need to put in the balance between sophistication and simplicity to think about the future maintenance!